Bugfix for issue #38

nmarshall23 opened the pull request:
JMOD was using Nashorn to parse Json. Now using Gson to parse the JMODInfo file.

I was wrong about the cause of this bug. In JRE 8u60 a feature was added to Nashorn to parse json into  Java Objects.

SvenKayser committed:
Indev commit 161020-1

Commit:
Bugfix for issue #38
JMOD was using Nashorn to parse Json. Now using Gson to parse Json.

SvenKayser commented:
Well I'm not sure if I want that.

The difference between Nashorn parsing and Gson parsing is that Nashorn parses actually a "true" javascript file that allows comments, while Gson does not. Because, interestingly: It used to be done with Gson before. I'll have to give this some thought.

SvenKayser committed:
Merge pull request #45 from SvenKayser/master

Master to indev merge

SvenKayser committed:
Indev Commit

SvenKayser committed:
Merge branch 'indev' of https://github.com/jeffpengMC/JMOD into indev

SvenKayser committed:
indev 1.3.1-1

SvenKayser committed:
indev 1.3.1-2
- Implemented Plugin System (WIP)
- Started singeling out mod related code into modules

SvenKayser committed:
Cleanup

SvenKayser committed:
Submodularization (if that's a word)

SvenKayser committed:
Cleanup

SvenKayser committed:
Cleanup

SvenKayser committed:
Updated Submodules

SvenKayser committed:
indev 1.3.1-3

SvenKayser committed:
indev 1.3.1-3 Cleanup1

SvenKayser committed:
The Big Clean Up

SvenKayser committed:
The Big Cleanup 3

SvenKayser committed:
The Big Clean Up 4

SvenKayser committed:
Added JMOD Plugin Template

SvenKayser committed:
The Big Clean Up 5

SvenKayser committed:
indev 1.3.1-6

SvenKayser committed:
indev 1.3.1-7

SvenKayser committed:
indev 1.3.1-8

SvenKayser committed:
indev 1.3.1-9

SvenKayser committed:
Merge branch 'indev' into bugfix/38.Nashorn-bug

SvenKayser commented:
Went into indev as I cannot merge this into master without further building upon this as it would break most jmods reteo is using.
